### PR TITLE
[#129][BUG] Extra space after Important Academic events on calendar page

### LINKED ISSUE #129: [Bug] Empty space below  Important Academic Events in the calendar
There should not be extra space below when there is no event in the Important Academic Events category
image

fixed the issue:
Screenshot 2023-02-14 at 8 29 43 AM

Screenshot 2023-02-14 at 8 31 40 AM
Screenshot 2023-02-14 at 8 32 10 AM

### PR DESCRIPTION
**Pull request by kikkiii21**
- removed the empty space under Important Academic Event category
- added padding at the bottom
- added generic font name to remove a warning

This closes #129

### CODE REVIEW

**Review by EyalAzimov — APPROVED** [review]
Thanks!

File: src/components/Calendar/Calendar.js [hunk 1]
```diff
@@ -219,7 +219,7 @@ export default function CalendarView() {
     const academicEventsDisplay = (
       <> <EventCard justifyContent='auto' width='92vw' height='30px' marginTop='5px' overflow='initial'
         content={<EventHeader content={"Important Academic Events"}/>}  backgroundColor='#E5A712' />
-       <div className="events">
+       <div className="events" style={{overflowY:'auto'}}>
```
Comment (mnashat) [inline src/components/Calendar/Calendar.js:222]: Why is overflowY set here when it is already set for the events css class? What does that do?

Comment (kikkiii21) [reply src/components/Calendar/Calendar.js:222]: apparently nothing. I removed it and behaviour seems the same

**Review by mnashat — APPROVED** [review]

**Comment by mnashat** [comment]
Good job!

**Pull request merged**